packages/neo-one-website/src/components/reference/Select.tsx

```diff
@@ -0,0 +1,29 @@
+// tslint:disable no-any match-default-export-name
```

Comment:
Remove this and move the `Select` component from `neo-one-developer-tools-frame` to `neo-one-react-common`